realm/realm-library/src/androidTest/java/io/realm/RealmMigrationTests.java

```diff
@@ -694,9 +694,24 @@ public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
     public void openPreNullWithRequired() throws IOException {
         configFactory.copyRealmFromAssets(context,
                 "default-before-migration.realm", Realm.DEFAULT_REALM_NAME);
+        RealmMigration migration = new RealmMigration() {
```

Comment:
I think you can actually use `NullTypes` or `NoPrimaryKeyNullTypes` for testing. no need to change the `AllTypes`, then no need to update this binary file.

Also, you can add test for nullable fields which is missed right now.